Scripts/NodeDataCache.cs

```diff
@@ -56,10 +56,13 @@ public static class NodeDataCache {
             } else {
                 // Else, check all relevant DDLs (slower)
                 // ignore all unity related assemblies
+                // never ignore current executing assembly
                 foreach (Assembly assembly in assemblies) {
-                    if (assembly.FullName.StartsWith("Unity")) continue;
-                    // unity created assemblies always have version 0.0.0
-                    if (!assembly.FullName.Contains("Version=0.0.0")) continue;
+                    if(assembly != Assembly.GetExecutingAssembly()) {
```

Comment:
Perhaps we should cache the executing assembly so we don't have to get it once for every assembly we check?

Reply:
Sure. Done.